proteobench/modules/dda_quant/module.py

```diff
@@ -262,6 +262,9 @@ def load_input_file(self, input_csv: str, input_format: str) -> pd.DataFrame:
                 lambda x: aggregate_modification_column(x.sequence, x.modifications),
                 axis=1,
             )
+        elif input_format == "i2MassChroQ":
+            input_data_frame = pd.read_csv(input_csv, low_memory=False, sep="\t")
+            input_data_frame["proforma"] = input_data_frame["ProForma"]
```

Comment:
Hey! Robbin, I don't understand something. Line 5 of the toml, we indicate that the proforma modified sequences are in the column "ProForma". Does it make sense to have to specify it here differently here?